be/src/olap/tablet.cpp

```diff
@@ -839,6 +839,30 @@ bool Tablet::check_path(const std::string& path_to_check) {
             return true;
         }
     }
+    for (auto& inc_version_rowset : _inc_rs_version_map) {
+        bool ret = inc_version_rowset.second->check_path(path_to_check);
+        if (ret) {
+            return true;
+        }
+    }
+    return false;
+}
+
+bool Tablet::check_rowset_id(RowsetId rowset_id) {
+    bool ret = false;
```

Comment:
_rs_version_map will be modified when you call this function.
A readlock is necessary.